rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -100,6 +112,9 @@ def _load_count_vect_params(self):
         # if convert all characters to lowercase
         self.lowercase = self.component_config["lowercase"]
 
+        # Flag to check if corresponding count vectorizers for each attribute of a message is trainable or not
+        self.is_trained = {attribute: False for attribute in MESSAGE_ATTRIBUTES}
```

Comment:
what do you mean is trainable?

Reply:
Any of the three countvectorizers could possible throw a ValueError during training and this needs to be recorded to persist the vocabulary of the models properly. They would throw a value error when vocabulary of passed text is empty, which could happen if there are no response selector examples and the text for response attribute of all messages is empty.